samples/nrf_rpc/protocols_serialization/client/snippets/crash_gen/snippet.yml

```diff
@@ -0,0 +1,9 @@
+#
```

Comment:
Don't you want to enable this for `ci` test case in `sample.yml`?

Reply:
I will do it in a separate PR